### PR TITLE
[devtools-translate] Chinese - What's New in DevTools (Chrome 101)

### PR DESCRIPTION
**Pull request by liuliangsir**
Fixes: https://github.com/GoogleChrome/developer.chrome.com/issues/2580

This PR adds Simplified Chinese translations for "What's New In DevTools (Chrome 101)". https://developer.chrome.com/blog/new-in-devtools-101/

### CODE REVIEW

File: site/zh/blog/new-in-devtools-101/index.md [hunk 1]
```diff
@@ -15,98 +15,128 @@ tags:
...
 
 {% Video src="video/dPDCek3EhZgLQPGtEG3y0fTn4v82/Jy7NEDZs6XJb90EWqETj.mp4", class="screenshot", autoplay=true, controls=true, loop=true, muted=true %}
 
 <!-- Apart from that, you can export the recording as well. After [recording a user flow](/docs/devtools/recorder/#record), click on the export button. There are 3 export options: -->
+除了上面所说的导入用户流程记录，您还可以导出这些记录。在[录制完用户流程](/docs/devtools/recorder/#record)之后，点击导出按钮。会弹出 3 个导出选项：
 
 <!-- - **Export as a JSON file**. Download the recording as a JSON file. -->
+- **导出为 JSON 文件**。下载的记录为一个 JSON 文件。
 <!-- - **Export as a @puppeteer/replay script**. Download the recording as a [Puppeteer Replay](https://github.com/puppeteer/replay) script.  -->
+- **导出为 @puppeteer/replay 脚本**。下载的记录为一段 [Puppeteer Replay](https://github.com/puppeteer/replay）脚本
```
Comment (hanselfmu) [inline site/zh/blog/new-in-devtools-101/index.md:39]: Missing a `。` in the end.

**Review by hanselfmu — APPROVED** [review]
LGTM, thanks!

File: site/zh/blog/new-in-devtools-101/index.md [hunk 2]
```diff
@@ -15,98 +15,128 @@ tags:
 draft: true
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-101/index.md:15]:
```suggestion
```

**Review by jecfish — APPROVED** [review]

File: site/zh/blog/new-in-devtools-101/index.md [hunk 3]
```diff
@@ -5,108 +5,137 @@ authors:
...
   - new-in-devtools
   - devtools
   - chrome-101
-draft: true
 ---
 
-*感谢[韩昌浩 @hanselfmu](https://github.com/hanselfmu) 提供的翻译*。
+*感谢 [流浪大法师 @liuliangsir](https://github.com/liuliangsir) 提供的翻译*。
 
 {% include 'partials/devtools/zh/banner.md' %}
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-101/index.md:20]:
```suggestion

{% YouTube id='u9GRAliBrM8' %}
```

**Pull request merged**